tests/models/xlm_roberta/test_modeling_xlm_roberta.py

```diff
@@ -49,6 +50,14 @@ def test_xlm_roberta_base(self):
         # compare the actual values for a slice of last dim
         self.assertTrue(torch.allclose(output[:, :, -1], expected_output_values_last_dim, atol=1e-3))
 
+        if is_torch_sdpa_available():
+            model = XLMRobertaModel.from_pretrained("FacebookAI/xlm-roberta-base", attn_implementation="sdpa")
+            with torch.no_grad():
+                output_sdpa = model(input_ids)["last_hidden_state"].detach()
+            self.assertEqual(output.shape, expected_output_shape)
+            # compare the actual values for a slice of last dim
+            self.assertTrue(torch.allclose(output, output_sdpa, atol=1e-3))
```

Comment:
Let's just put this in a separate `test_xlm_roberta_base_sdpa` test and use `self.skipTest` if `is_torch_sdpa_available()` evaluates to False